src/patterns/components/usa-banner/_usa-banner.scss

```diff
@@ -1,13 +1,9 @@
 @use "sass:list";
 @use "sass:map";
-
-@use "../core" as *;
+@use "../../stylesheets/core" as *;
 
 $banner-guidance-measure: 3;
-$banner-icon-colors: get-link-tokens-from-bg(
-  $theme-banner-background-color,
-  $theme-banner-link-color
-);
+$banner-icon-colors: get-link-tokens-from-bg($theme-banner-background-color, $theme-banner-link-color);
```

Comment:
Did you re-format these or did you use `prettier:scss`

Reply:
I reformatted it, I haven't circled back around to the prettier stuff yet.